src/math-functions.md

```diff
@@ -108,3 +108,10 @@
    - Second parameter denotes number of digits after floating point
      
      <img src="images/round1.png" width="200" />
+
+16. **Random**
+   - Declare integers x, y
+   - Take input for x, y  
```

Comment:
```suggestion
   - Take input or assign value for y  
```

I understand that the other functions' definitions have similar typos, but it would be better to fix at least the newer ones.